deployments/chef/CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 # Changelog
 
+## chef-v0.15.0
+
+- Breaking Change: The default for the `auto_instrumentation_otlp_endpoint` option has been changed from
+  `http://127.0.0.1:4317` to `''` (empty), i.e. defer to the default `OTEL_EXPORTER_OTLP_ENDPOINT` value for each
+  activated SDK.
+- Add support for the `auto_instrumentation_otlp_endpoint_protocol` and `auto_instrumentation_metrics_exporter` options
+  to configure the `OTEL_EXPORTER_OTLP_PROTOCOL` and `OTEL_METRICS_EXPORTER` environment variables, respectively.
+
```

Comment:
Java 2.x also enables metrics and logs by default. Is that a breaking change worth mentioning as well?

Reply:
> Java 2.x also enables metrics and logs by default. Is that a breaking change worth mentioning as well?

Should I just add a reference to https://github.com/signalfx/splunk-otel-java/releases/tag/v2.0.0-alpha?

Reply:
No, let's go.